cfme/tests/containers/test_one_image_analysis.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import toolbar as tb, tabstrip as tabs
+from cfme.containers import image
+from utils.wait import wait_for, TimedOutError
+from cfme.configure import tasks
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+# CMP-10064
+
+def test_image_analysis_finished():
+    image.run_smart_state_analysis()
+    is_task_finished()
+
+
+def is_task_finished():
```

Comment:
I assume the issue is the tab name `All VM Analysis Tasks` X `All VM and Container Analysis Tasks`, please check it out and feel free to make that fix a part of this PR.